ACE/tests/Recursive_Condition_Test.cpp

```diff
@@ -60,7 +59,8 @@ class Test_Handler : public ACE_Event_Handler
 
 // These are for the basic functionality tests.
 ACE_SYNCH_RECURSIVE_MUTEX mutex_;
-ACE_Condition<ACE_SYNCH_RECURSIVE_MUTEX> condition_ (mutex_);
+//ACE_Condition<ACE_SYNCH_RECURSIVE_MUTEX> condition_ (mutex_);
```

Comment:
I thought I'd leave that in as it could be used to quickly test whether the compiler/linker is really picking up the template specialization (instead of the generic implementation)